src/test/java/com/thealgorithms/searches/Rotated_Binary_Search.java

```diff
@@ -0,0 +1,57 @@
+// This is returning the index of that element
+class Rotated_Binary_Search{
+  public static void main(String[] args) {
+  
+        int[] arr = { 4, 5, 6, 7, 0, 1, 2 };
+        System.out.println(search(arr,5));
+    }
+
+    static int search(int[] nums, int target) {
+        int pivot = findPivot(nums);
+        if(pivot ==-1){
+            return Binarysearch(nums, target, 0, nums.length-1);
+        }
+        if(nums[pivot]==target){
+            return pivot;
+        }
+        if(target>=nums[0]){
+            return Binarysearch(nums, target, 0, pivot-1);
+        }
+        return Binarysearch(nums, target, pivot+1, nums.length-1);
+    }
+
+    static int Binarysearch(int[] arr, int target, int start, int end) {
```

Comment:
This function name should follow camel case.'B' should be small.

Reply:
A 's' should be capital.